evap/evaluation/models.py

```diff
@@ -1621,7 +1621,13 @@ class UserProfile(AbstractBaseUser, PermissionsMixin):
 
     title = models.CharField(max_length=255, blank=True, default="", verbose_name=_("Title"))
     first_name_given = models.CharField(max_length=255, blank=True, verbose_name=_("given first name"))
-    first_name_chosen = models.CharField(max_length=255, blank=True, default="", verbose_name=_("display name"))
+    first_name_chosen = models.CharField(
+        max_length=255,
+        blank=True,
+        default="",
+        verbose_name=_("display name"),
+        help_text=_("This will replace your given first name."),
```

Comment:
"given first name" is a bit odd, maybe it should be "given name" instead?

Reply:
We decided that the form field should be named "given first name" to make clear that it is about the first name. "Given name" might be ambiguous.